desktop/windows/index.md

```diff
@@ -273,9 +273,30 @@ For more information about using the Kubernetes integration with Docker Desktop,
 
 ### Software Updates
 
-The Software Updates section notifies you of any updates available to Docker Desktop. You can choose to download the update right away, or click the **Release Notes** option to learn what's included in the updated version.
+The **Software Updates** section notifies you of any updates available to Docker Desktop. When there's a new update, you can choose to download the update right away, or click the **Release Notes** option to learn what's included in the updated version.
 
-If you are on a Docker Team or a Business subscription, you can turn off the check for updates by clearing the **Automatically Check for Updates** checkbox in the [General](#general) settings. This will also disable the notification badge that appears on the Docker Dashboard.
+> **Disable automatic check for updates**
+>
+> Starting with Docker Desktop 4.2.0, the option to turn off the automatic check for updates is available for users on all Docker subscriptions, including Docker Personal. Update to [Docker Desktop 4.2.0](../windows/release-notes/index.md) to start using this feature.
+{: .important}
+
+Turn off the check for updates by clearing the **Automatically Check for Updates** check box. This disables notifications in the Docker menu and also the notification badge that appears on the Docker Dashboard. To check for updates manually, select the **Check for updates** option in the Docker menu.
+
+To allow Docker Desktop to automatically download new updates in the background, select **Always download updates**. This downloads newer versions of Docker Desktop when an update becomes available.
+
+Select **Use nightly builds** to download nightly builds instead of stable builds When you check for available updates.
+
+After selecting your software update preferences, click **Apply and Restart** for the changes to take effect.
+
+## Dashboard
+
+The Docker Desktop Dashboard enables you to interact with containers and applications and manage the lifecycle of your applications directly from your machine. The Dashboard UI shows all running, stopped, and started containers with their state. It provides an intuitive interface to perform common actions to inspect and manage containers and existing Docker Compose applications. For more information, see [Docker Desktop Dashboard](../dashboard.md).
```

Comment:
nit: is this not the `Docker Dashboard`?
```suggestion
The Docker Dashboard enables you to interact with containers and applications and manage the life-cycle of your applications directly from your machine. The Dashboard UI shows all running, stopped, and started containers with their state. It provides an intuitive interface to perform common actions to inspect and manage containers and existing Docker Compose applications. For more information, see [Docker Dashboard](../dashboard.md).
```

Reply:
Yes, it's Docker Dashboard, but lifecycle is one word.

Reply:
Good catch. Updated to Docker Dashboard